llama_index/indices/postprocessor/types.py

```diff
@@ -33,10 +33,25 @@ def _get_prompt_modules(self) -> PromptMixinType:
     def class_name(cls) -> str:
         return "BaseNodePostprocessor"
 
-    @abstractmethod
     def postprocess_nodes(
         self,
         nodes: List[NodeWithScore],
         query_bundle: Optional[QueryBundle] = None,
+        query_str: Optional[str] = None,
```

Comment:
nit: should we just use single param, to be consistent with e.g. retriever and query engine:

```
    def retrieve(self, str_or_query_bundle: QueryType) -> List[NodeWithScore]:
```

Reply:
oh gotcha, kk I guess this could work.